lib/rack/honeycomb/middleware.rb

```diff
@@ -2,8 +2,14 @@
 
 module Rack
   module Honeycomb
+    # Prefix for attaching arbitrary metadata to the `env`. Will be deleted from
+    # from the `env` once it's pulled off of the `env` and onto a Honeycomb
+    # event.
+    HEADER_PREFIX = "honeycomb_"
```

Comment:
How about `ENV_PREFIX` rather than `HEADER_PREFIX`?

I think `honeycomb.` would be slightly more idiomatic (and thereby slightly less likely to be collided with), but there's not much in it, and you'd need to change the regex to `#{Regex.escape prefix}`.